Gems/PhysX/Code/Editor/Source/Components/Conversion/CollidersPrefabConversion.cpp

```diff
@@ -23,11 +24,13 @@ namespace PhysX
 {
     void UpdatePrefabsWithColliderComponents(const AZ::ConsoleCommandContainer& commandArgs);
 
+    // O3DE_DEPRECATION_NOTICE(GHI-14718)
     AZ_CONSOLEFREEFUNC(
         "ed_physxUpdatePrefabsWithColliderComponents",
         UpdatePrefabsWithColliderComponents,
         AZ::ConsoleFunctorFlags::Null,
-        "Finds entities with collider components and no rigid bodies and updates them to the new pattern which requires a static rigid body component.");
+        "Finds entities with collider components and no rigid bodies and updates them to the new pattern which requires a static rigid body component. "
+        "Finds entities with collider components using physx asset and replace it with a mesh collider component.");
```

Comment:
Quick question: Does this command now just do both operations in one?

Reply:
Yes, this is intentional so in the next o3de release it's only 1 command we ask users to run in the release notes.

Reply:
Understood perfect, and is there a warning/pop-up that we can present users with when opening the level to let them know they need to run this command?

Reply:
At the moment we don't have a way to detect if a prefab needs to be updated or not I'm afraid, but that feature is becoming more and more important and it'd be ideal if the editor will just tell customers when they have to do some process to update their content to work with newer versions of the engine, but that support doesn't exist at the moment. 

It's worth mentioning I'm updating the most important repos in o3de, so all the content in our repos should be updated, it's only for user's own prefabs they will have to run it.

Reply:
I see 🙁 That is a real shame as just having a way to detect and then prompt the user would be incredibly helpful and guard against people not understanding why their level won't open.

Understood, I just think as time goes by that number is going to increase significantly and we should really think about addressing it before then.

Tagging @srikappa-amzn, @monroegm and @forkercat for visibility and I wonder if/when @o3de/sig-content might be able to add support for this.

Reply:
question: What happens if a user tries to load a prefab with the old data? Will they still be able to load the rest of the prefab except for the entities with the old collider components?  If that is not the case, we should not be pushing this change in its current state imo.

suggestion: Looking at the code in Gems\PhysX\Code\Editor\Source\Components\Conversion\PrefabConversionUtils.cpp , it looks like we are directly making changes to the Prefab DOM data. If that's the case, it shouldn't be hard to provide a notification when a prefab is loaded in the editor (whether it is through loading the level or instantiating a prefab). And systems like PhysX can listen to that notification and execute the same code in UpdatePrefabPhysXColliders(). Then you'll have the ability to either force update user's data or ask them for their permission or just show them a warning.